src/http_util/error.rs

```diff
@@ -57,6 +57,19 @@ pub fn parse_error_kind(err: &isahc::Error) -> ErrorKind {
     }
 }
 
+/// parse_http_error_code will parse HTTP status code into `ErrorKind`
+pub fn parse_http_error_code(code: StatusCode) -> ErrorKind {
```

Comment:
How about renaming to `parse_error_status_code`?

Reply:
Good, there are no such "error codes" but only status codes in HTTP.